app/javascript/dashboard/components/ui/Dropdown/DropdownListItemButton.vue

```diff
@@ -19,6 +27,12 @@ defineProps({
     @focus="$emit('focus')"
   >
     <div class="inline-flex items-center gap-3 overflow-hidden">
+      <fluent-icon
+        v-if="leftIcon"
+        icon="status"
```

Comment:
```suggestion
        icon="status"
```

Is this okay to be hardcoded?